cmd/lassie/fetch.go

```diff
@@ -78,9 +85,14 @@ func Fetch(c *cli.Context) error {
 	timeout := c.Duration("timeout")
 	timeoutOpt := lassie.WithProviderTimeout(timeout)
 
-	var opts = []lassie.LassieOption{timeoutOpt}
-	if fetchProviderAddrInfo != nil {
-		finderOpt := lassie.WithFinder(explicitCandidateFinder{provider: *fetchProviderAddrInfo})
+	host, err := libp2p.New(libp2p.ResourceManager(&network.NullResourceManager{}))
```

Comment:
what's this about?

Reply:
we've got a bit of a weird dependency hell here -- we need the libp2p host for the finder, but we need the finder before we have a Lassie, so lassie can't build it. I tried internal.InitHost but it's inside the pkg dir now :P